provisionersdk/scripts/bootstrap_windows.ps1

```diff
@@ -1,6 +1,6 @@
 # Sleep for a while in case the underlying provider deletes the resource on error.
 trap {
-	Write-Error '=== Agent script exited with non-zero code. Sleeping 24h to preserve logs...'
+	Write-Error "=== Agent script exited with non-zero code. Sleeping 24h to preserve logs..."
```

Comment:
I'm curious how folks would be using `sh -c` on Windows, but I suppose it's good to remain consistent 👍